catalyst_voices/packages/catalyst_voices_shared/lib/src/platform_widget_builder/platform_widget_builder.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:catalyst_voices_shared/src/platform/catalyst_platform.dart';
+import 'package:flutter/widgets.dart';
+
+class PlatformWidgetBuilder<T> extends StatelessWidget {
```

Comment:
Let's add documentation with an example of how to use it.
[Here is an excellent example of documentation.](https://github.com/flutter/flutter/blob/b2575d54892416fa9055d6cfe6f67826604aacce/packages/flutter/lib/src/material/text_form_field.dart#L52)